### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by techbelle**
adding some basics to our repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+## Apache Arrow
+We utilize Apache Arrow to offer fast and easy conversion of MongoDB query result sets to multiple numerical data formats popular among developers including NumPy ndarrays, Pandas DataFrames, parquet files, csv, and more.
+
+We chose Arrow for this because of its unique set of characteristics:
+- language-independent 
+- columnar memory format for flat and hierarchical data, 
+- organized for efficient analytic operations on modern hardware like CPUs and GPUs
+- zero-copy reads for lightning-fast data access without serialization overhead
+it was simple and fast, and from our perspective Apache Arrow is ideal for processing and transport of large datasets in high-performance applications.
```
Comment (ShaneHarvey) [inline README.md:12]: missing `-` on the last line?

Comment (blink1073) [inline README.md:12]:
```suggestion
  - it was simple and fast, and from our perspective Apache Arrow is ideal for processing and transport of large datasets in high-performance applications.
```

Comment (blink1073) [reply README.md]: Updated

**Review by ShaneHarvey — COMMENTED** [review]
Should we update to mention auto-schema discovery instead of manual to simplify the example?

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**